tests/installation/bootloader.pm

```diff
@@ -22,15 +22,19 @@ use lockapi 'mutex_wait';
 use bootloader_setup;
 use bootloader_spvm;
 use registration;
-use version_utils ':SCENARIO';
+use version_utils qw(:VERSION :SCENARIO);
 use utils;
+use Utils::Architectures qw(is_i586 is_i686);
 use Utils::Backends 'is_spvm';
 
 # hint: press shift-f10 trice for highest debug level
 sub run {
     return boot_spvm if is_spvm;
     return           if pre_bootmenu_setup == 3;
     return           if select_bootmenu_option == 3;
+    my $boot_cmd = 'ret';
+    # Tumbleweed livecd has been switched to grub with kiwi 9.17.41 except 32bit
+    uefi_bootmenu_params() if (is_livecd && is_tumbleweed && !is_i586 && !is_i686);
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You could negate the logic check but I guess I am about to give up anyway on this suggestion…

Reply:
I'm not sure I get your point here, the change here is still considering Tumbleweed as the default product, but having an *exception behavior* on Tumbleweed's livecd(non-UEFI).

Reply:
Yeah, sure. I suggest to try to always avoid `is_tumbleweed` as this will only be a temporary measure until any next version of SLE or Leap also run into the same problem. When one writes instead of `is_tumbleweed` something like `!is_leap('<15.2') && !is_sle('<15-SP2')` then most likely any new version of Leap and SLE will not run into the same problem.